packages/resin-event-log/src/resin-event-log.js

```diff
@@ -61,14 +61,10 @@ module.exports = function(options) {
 		userId: null,
 		prefix: prefix,
 		start: function(user, callback) {
-			if (!user) {
-				return Promise.reject(new Error(
-					'user is required to start events interaction.'
-				)).asCallback(callback)
```

Comment:
This shouldn't go away totally, it should just move into the mixpanel/ga adapters, so those fail individually instead. That way this will be usable if you only have goSquared configured, and we can look at how to transparently support mixpanel & go squared at the same time separately later.

Right now, if you do try to call .login on those adapters without a user, as this PR does, I suspect you'll break things... Mixpanel for example will define & alias for a non-existent user id, which probably break all sorts. Testing welcome :smiley: